lib/phoenix/code_reloader.ex

```diff
@@ -14,15 +14,24 @@ defmodule Phoenix.CodeReloader do
 
   @doc """
   Reloads code for the current Mix project by invoking the
-  `:reloadable_compilers`.
+  `:reloadable_compilers` on the list of `:reloadable_apps`.
 
   This is configured in your application environment like:
 
       config :your_app, YourApp.Endpoint,
-        reloadable_compilers: [:gettext, :phoenix, :elixir]
+        reloadable_compilers: [:gettext, :phoenix, :elixir],
+        reloadable_apps: [:ui, :backend]
 
   Keep in mind `:reloadable_compilers` must be a subset of the
   `:compilers` specified in `project/0` in your `mix.exs`.
+
+  The `:reloadable_apps` defaults to `nil`. In such case
```

Comment:
Can we add a link to this function and to the `Phoenix.CodeReloader` when describing the `code_reloader` option in `lib/phoenix/endpoint.ex`? Otherwise I think this is quite hard to reach.

Reply:
Makes sense. Added it here: https://github.com/phoenixframework/phoenix/pull/2887/files#diff-2e48d48e1c6c3b085dab25613ade742bR77